sdks/js/packages/core/react/components/organization/routes.tsx

```diff
@@ -0,0 +1,318 @@
+import { useCallback, useEffect } from 'react';
+import {
+  Outlet,
+  createRoute,
+  createRootRouteWithContext,
+  useRouteContext,
+  RouteComponent
+} from '@tanstack/react-router';
+
+import { Flex } from '@raystack/apsara';
+
+import { Toaster } from 'sonner';
+import { useFrontier } from '~/react/contexts/FrontierContext';
+import Domain from './domain';
+import { AddDomain } from './domain/add-domain';
+import { VerifyDomain } from './domain/verify-domain';
+import GeneralSetting from './general';
+import { DeleteOrganization } from './general/delete';
+import WorkspaceMembers from './members';
+import { InviteMember } from './members/invite';
+import UserPreferences from './preferences';
+
+import { default as WorkspaceProjects } from './project';
+import { AddProject } from './project/add';
+import { DeleteProject } from './project/delete';
+import { ProjectPage } from './project/project';
+
+import WorkspaceSecurity from './security';
+import { Sidebar } from './sidebar';
+import WorkspaceTeams from './teams';
+import { AddTeam } from './teams/add';
+import { DeleteTeam } from './teams/delete';
+import { TeamPage } from './teams/team';
+import { UserSetting } from './user';
+import { SkeletonTheme } from 'react-loading-skeleton';
+import { InviteTeamMembers } from './teams/members/invite';
+import { DeleteDomain } from './domain/delete';
+import Billing from './billing';
+import Tokens from './tokens';
+import { ConfirmCycleSwitch } from './billing/cycle-switch';
+import Plans from './plans';
+import ConfirmPlanChange from './plans/confirm-change';
+
+export interface CustomComponent {
+  name: string;
+  path: string;
+  category: 'Organization' | 'User';
+  component: RouteComponent;
+}
+
+export interface OrganizationProfileProps {
+  organizationId: string;
+  defaultRoute?: string;
+  showBilling?: boolean;
+  showTokens?: boolean;
+  showPreferences?: boolean;
+  hideToast?: boolean;
+  customComponents?: CustomComponent[];
```

Comment:
Should we call it `customPaths`, `customPages` or `customScreens` as in future we are planning to make this full screen instead of popup?